subgraphs/hop-protocol/src/common/getters.ts

```diff
@@ -0,0 +1,249 @@
+// import { log } from "@graphprotocol/graph-ts"
```

Comment:
I don't think you need any of the files in `common` if you are using the sdk, right?